### PR TITLE
Ensure code coverage covers all modules in Azure Communication Services Phone Numbers.

### PR DESCRIPTION
**Pull request by whisper6284**
# Description

Ensure code coverage covers all modules in Azure Communication Services Phone Numbers.

### CODE REVIEW

File: sdk/communication/azure-communication-phonenumbers/pom.xml [hunk 1]
```diff
@@ -134,7 +134,15 @@
         <configuration>
           <includes>
             <include>com/azure/communication/phonenumbers/*.class</include>
+            <include>com/azure/communication/phonenumbers/implementation/*.class</include>
+            <include>com/azure/communication/phonenumbers/implementation/converters/*.class</include>
+            <include>com/azure/communication/phonenumbers/implementation/models/*.class</include>
+            <include>com/azure/communication/phonenumbers/models/*.class</include>
             <include>com/azure/communication/phonenumbers/siprouting/*.class</include>
+            <include>com/azure/communication/phonenumbers/siprouting/implementation/*.class</include>
+            <include>com/azure/communication/phonenumbers/siprouting/implementation/converters/*.class</include>
```
Comment (sofiar-msft) [inline sdk/communication/azure-communication-phonenumbers/pom.xml:143]: nit: these should be in line with the other includes

Comment (whisper6284) [reply sdk/communication/azure-communication-phonenumbers/pom.xml:143]: Sorry, I don't understand what you mean?

**Review by sofiar-msft — APPROVED** [review]

**Pull request merged**